### ISSUE TITLE
dyld: Symbol not found: __T0s19_emptyStringStorages6UInt32Vv

### ISSUE THREAD

**Issue by rwv** [vulgo/webdriver.sh#10]
I encountered the `dyld: Symbol not found:` 
```
$ webdriver --list
Password:
dyld: Symbol not found: __T0s19_emptyStringStorages6UInt32Vv
  Referenced from: /usr/local/etc/webdriver.sh/clover-patcher
  Expected in: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx/libswiftCore.dylib
 in /usr/local/etc/webdriver.sh/clover-patcher
/usr/local/bin/webdriver: line 235:  8526 Abort trap: 6           "${HOST_PREFIX}${LIBEXEC}${1}"
```

Here's my [libswiftCore.dylib.zip](https://github.com/vulgo/webdriver.sh/files/1890391/libswiftCore.dylib.zip)

I updated my Xcode but still encounters the same issue and my swift version is
```
$ xcrun swift -version
Apple Swift version 4.1 (swiftlang-902.0.48 clang-902.0.37.1)
Target: x86_64-apple-darwin17.4.0
```

I am on:
* Clover version: 4278
* macOS 10.13.3 (17D102)
* Web Driver 387.10.10.10.25.161

Thanks!

**Comment by vulgo** [comment]
Thanks for this, should be fixed in 1.4.1.

**Comment by rwv** [comment]
Thanks, problem is solved.
